compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrDeclarationStorage.kt

```diff
@@ -53,6 +51,9 @@ class Fir2IrDeclarationStorage(
     conversionScope: Fir2IrConversionScope,
     fakeOverrideMode: FakeOverrideMode
 ) : Fir2IrComponents by components {
+
+    internal var annotationGenerator: AnnotationGenerator? = null
```

Comment:
use "lateinit" instead of initializing with null and then null-check later?

Reply:
Yeah, I tried that as well; Alas, for built-in classes, this storage is used even before the visitor is created (which is used to initialize `AnnotationGenerator`).